### ISSUE TITLE
acl is not a valid value (aws_s3_bucket_invalid_acl)

### ISSUE THREAD

**Issue by netflash** [wata727/tflint#353]
Hi there,

My s3 bucket has the following config set

```
resource "aws_s3_bucket" "somebucket" {
...
acl           = "log-delivery-write"
...
}
```

tflint 0.9.0 treats this as an error

```
ERROR:4 acl is not a valid value (aws_s3_bucket_invalid_acl)
```

Looks like there's a missing item in array in this file

https://github.com/wata727/tflint/blob/7c303fa9e6a56b6dd8f9a9fc9bfcb21a1b7d3753/rules/awsrules/models/aws_s3_bucket_invalid_acl.go#L25-L29

You can find the full list of canned acl here - https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl

Let me know if you want me to create a PR.

PS 0.8.3 doesn't show this error.

**Comment by ineffyble** [comment]
I was the author of the original rule from 0.8.3, and I used the list you linked to.

I believe the issue is that the rule was replaced with a list of canned ACLs from the AWS SDK, which is actually incomplete.

So this is one case where the SDK is wrong, unfortunately.

**Comment by wata727** [comment]
As @ineffyble says, it seems to be an issue with the sdk rule generation. Now, @ricoli is opening an issue in aws-sdk. See https://github.com/wata727/tflint/pull/341#discussion_r300335909

I'm waiting for the aws-sdk team to respond, but I'm preparing a patch to fix this problem temporarily. I will release v0.9.1 soon.

**Comment by wata727** [comment]
v0.9.1 has been released

**Comment by netflash** [comment]
Thank you!

**Comment by ricoli** [comment]
thank you very much for the quick turnaround on this @wata727 , and hopefully the s3 team will keep the API model updated going forward as per https://github.com/aws/aws-sdk-go/issues/2683#issuecomment-509426997 :crossed_fingers:
